### ISSUE TITLE
Long-recording ASR recipe example

### ISSUE THREAD

**Issue by pzelasko** [pzelasko/lhotse#26]
I'm thinking that the next step (after merging #24) is to create an example data preparation recipe for something conversational (e.g. SWBD), where we have 10 mins or more of continuous recordings. So far, I only tested the cut mechanism on LibriMix, where the recordings are already cut into short utterances, so the 'cutting' procedure was more simplistic than in a typical scenario we want to support. 

We'd create Cuts from the transcribed segments by "padding" them with the actual acoustic context. These cuts could have different length "bins", e.g. 5, 10, 20 seconds so that we can construct the batches from utterances of similar lengths (and maybe scale the batch size accordingly). We'd also mix in some noise, e.g. MUSAN. 

We'll need to provide the supervision for the ASR. Once K2 is ready, these will be K2 graphs. For now we can start with [PyChain](https://github.com/YiwenShaoStephen/pychain), or just provide a word/character sequence (e2e seq2seq style). We might actually tackle preparing the supervisions in a separate PR, as we might need to introduce bigger changes such as some kind of vocabulary manifest, etc.

**Comment by pzelasko** [comment]
This is covered by AMI; we'll figure out the supervision part soon when integrating with k2.
